central/processlisteningonport/datastore/datastore_impl_test.go

```diff
@@ -2343,65 +2346,19 @@ func (suite *PLOPDataStoreTestSuite) makeRandomPlops(nport int, nprocess int, np
 	return plops
 }
 
-func (suite *PLOPDataStoreTestSuite) TestAddPodUids() {
```

Comment:
It doesn't seem like this test is needed anymore, as it checks that null poduids can be updated, and there should not be any more null poduids.